lib/loaders/api/loader_without_authentication_factory.js

```diff
@@ -1,6 +1,7 @@
 // @ts-check
```

Comment:
cc @joeyAghion this is what we were initially talking about, you can now cache body and headers for unauthenticated requests if the loader is specified with `header: true`.